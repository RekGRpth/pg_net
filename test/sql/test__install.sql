create extension curl_worker cascade;
