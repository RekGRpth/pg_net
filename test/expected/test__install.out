create extension curl_worker cascade;
NOTICE:  installing required extension "uuid-ossp"
